### PR TITLE
Clean up inst/config/workflow

### PR DESCRIPTION
**Pull request by jwildfire**
# Summary

This PR migrates/deletes functions in the inst/config/workflow folder: 
- Creates `/R/tplyr_shift_chart.R`. This seems to deal with the dependency issues and fix the shift table, which now renders. Still need to do #53 before v1.0 to have full safetyGraphics functionality. 
- Migrate lab distribution module to `/R/mod_lab_distribution.R` and add documentation. 
- Leaving `/inst/config/workflow/safety_histogram_chart.R` for now. Will deal with that in #45 

# Test notes

Confirm that shift table and lab distribution work as expected in safetyGraphics and in the exported html. 

```
devtools::install_github("safetyGraphics/safetyCharts", ref="fix-43-50")
library(safetyCharts)
devtools::install_github("safetyGraphics/safetyGraphics", ref="reports-updates")
library(safetyGraphics)
safetyGraphics::safetyGraphicsApp()
```

### CODE REVIEW

**Comment by jwildfire** [comment]
Merge this after #52

File: inst/config/tplyr_shift.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 label: Shift Table {Tplyr} + {kable}
 type: html
 domain: multiple
-pacakage: Tplyr
+package: safetyCharts
 workflow:
```
Comment (elimillera) [inline inst/config/tplyr_shift.yaml:5]: Does this yaml need the export?

**Review by elimillera — APPROVED** [review]
One question but looks good

File: inst/config/tplyr_shift.yaml (hunk 1 above)
Comment (jwildfire) [reply inst/config/tplyr_shift.yaml:5]: Doesn't hurt to add it, but will default to TRUE if nothing is provided.

**Pull request merged**